common/src/configuration/global.rs

```diff
@@ -84,6 +84,7 @@ pub struct GlobalConfig {
     pub transaction_chain_monitoring_timeout: Duration,
     pub transaction_direct_send_timeout: Duration,
     pub transaction_broadcast_send_timeout: Duration,
+    pub dust_collector_enabled: bool,
```

Comment:
The global config struct is actually deprecated. There's a better config setup that @dunnock introduced, but iirc only the mempool config has migrated to that structure. Just FYI.